### PR TITLE
Update categories.csv

### PR DESCRIPTION
**Pull request by Dum4G**
Maybe it's relevant to add a new category for channels that aren't exactly covering news but spread analytics and visions about them that some may consider as biased, controversial or propagandistic. Inspired by https://github.com/iptv-org/database/issues/9508 and some RU channels

### CODE REVIEW

**Comment by BellezaEmporium** [comment]
This would mean the channel **only** axes their programmation on political aspects. Wouldn't that be biased, considering this is some analytics of news ?

**Comment by Dum4G** [comment]
Channels like this don't cover news in traditional sense, usually it's in the form of talkshows or "lives" where hosts share their opinion for the topic or ask for the guest opinion. It's exactly biased never-ending analytics

**Comment by sguinetti** [comment]
I agree with the creation of a category. If channels are categorized as "news" only, we would be assuming that they are unbiased and that they respect the parameters to combat the occasional fake news. I could name the category of "Opinion journalism" when the channel is only based on a clear editorial view of things. It is not clear to me to what extent it can be differentiated, perhaps neutrality is a relevant point. In Peru, the concept of "alternative press" is used for a position contrary to the officialism of the government in power. For investigative journalism, the concept of "independent press" is used.

**Comment by BellezaEmporium** [comment]
> I agree with the creation of a category. If channels are categorized as "news" only, we would be assuming that they are unbiased and that they respect the parameters to combat the occasional fake news. I could name the category of "Opinion journalism" when the channel is only based on a clear editorial view of things. It is not clear to me to what extent it can be differentiated, perhaps neutrality is a relevant point.
> 
> In Peru, the concept of "alternative press" is used for a position contrary to the officialism of the government in power. For investigative journalism, the concept of "independent press" is used.

I do agree with that statement. Then it would work on channels that are strictly related to politics and such. Ex : Tucker Carlson Network

**Comment by BellezaEmporium** [comment]
I think the "Politics" term is a bit too straight-forward. Maybe "Talks" should be a better idea. Englobes all kinds of talkshows.

**Comment by Dum4G** [comment]
Can Opinion-based category be described in one unified translatable word?

**Comment by BellezaEmporium** [comment]
> Can Opinion-based category be described in one unified translatable word?

Could it be "Debate" ?

**Comment by Dum4G** [comment]
Problem is some channels are mascing it's original intent, for example Tsargrad is religious channel and you can even see a pope running some personal shows, but the attitude of the channel is covering every news from Imperialistic conservative standpoint. So I would rather stick to politics so people could understand the channel is controversial

**Comment by sguinetti** [comment]
I think "Debate" is a somewhat ambiguous category. On my review, "Political" seems more accurate than "Opinions" because it is more specific. There are religious opinions, economic opinions, entertainment ("show business") and so on....

**Comment by sguinetti** [comment]
We would need more comments for this category and how we should proceed in some channels.

**Comment by sguinetti** [comment]
> I do agree with that statement. Then it would work on channels that are strictly related to politics and such. Ex : Tucker Carlson Network

In Peru, there is also a notable example, La Abeja channel.

**Comment by Alstruit** [comment]
How about "Podcast"?

**Comment by MapGuy11** [comment]
I'm divided on the point of us making this distinction. As I understand adding the category would clarify some channels, but the political can be very broad depending on the angle we are coming from. As you have some religious shows that have a political/party bias but is branded as a religious channel.

**Comment by sguinetti** [comment]
> I'm divided on the point of us making this distinction. As I understand adding the category would clarify some channels, but the political can be very broad depending on the angle we are coming from. As you have some religious shows that have a political/party bias but is branded as a religious channel. I feel like whatever we do wont be the best but that is kind of the genre.

I am not sure if the repository allows multiple categories such as "religious;political".

On the other hand, bacuse "Politics" would be an ambiguous category, I thought of using a category that represents this idea as "public-affairs". That's where podcasts, talks and others fit into that category. For some examples I taked from [Imdb page](https://www.imdb.com/search/keyword/?keywords=public-affairs) and [Wikipedia](https://en.wikipedia.org/wiki/Public_affairs_(broadcasting)). If you choose that category, I think it would be best for everyone.

**Comment by MapGuy11** [comment]
> > I'm divided on the point of us making this distinction. As I understand adding the category would clarify some channels, but the political can be very broad depending on the angle we are coming from. As you have some religious shows that have a political/party bias but is branded as a religious channel. I feel like whatever we do wont be the best but that is kind of the genre.
> 
> I am not sure if the repository allows multiple categories such as "religious;political".
> 
> On the other hand, bacuse "Politics" would be an ambiguous category to separate the news channels, I thought of using a category that represents this idea as "public-affairs". That's where podcasts, talks and others fit into that category. For some examples I taked from [Imdb page](https://www.imdb.com/search/keyword/?keywords=public-affairs) and [Wikipedia](https://en.wikipedia.org/wiki/Public_affairs_(broadcasting)). If you choose that category, I think it would be best for everyone.

I agree with this

File: data/categories.csv [hunk 1]
```diff
@@ -18,6 +18,7 @@ movies,Movies
 music,Music
 news,News
 outdoor,Outdoor
+politics,Politics
```
Comment (sguinetti) [inline data/categories.csv:21]:
```suggestion
public-affairs,Public affairs
```

Comment (sguinetti) [reply data/categories.csv:21]: I don't know if it is possible to use hyphens or spaces for two words, but I leave the proposal here.

**Review by sguinetti — CHANGES_REQUESTED** [review]

**Comment by Dum4G** [comment]
> > I'm divided on the point of us making this distinction. As I understand adding the category would clarify some channels, but the political can be very broad depending on the angle we are coming from. As you have some religious shows that have a political/party bias but is branded as a religious channel. I feel like whatever we do wont be the best but that is kind of the genre.
> 
> I am not sure if the repository allows multiple categories such as "religious;political".
> 
> On the other hand, bacuse "Politics" would be an ambiguous category to separate the news channels, I thought of using a category that represents this idea as "public-affairs". That's where podcasts, talks and others fit into that category. For some examples I taked from [Imdb page](https://www.imdb.com/search/keyword/?keywords=public-affairs) and [Wikipedia](https://en.wikipedia.org/wiki/Public_affairs_(broadcasting)). If you choose that category, I think it would be best for everyone.

In that case politics or public policy means that the broadcast is forced to exist and is sponsored by law or authorities, which is not always the case for channels we discuss. Just find me a word synonimous to politics, preferably a single one, that will perfectly describe the genre and can be understood across the world without googling or loosing it's sense

**Comment by AntiPontifex** [comment]
First of all, one of the biggest problems with this is how to be impartial: on what impartial authority are you using to claim it is propagandist? Secondly, I think this thread is looking for the term "government organ" as opposed to a legitimate news source. Perhaps, so-called questionable sources could be labelled with a warning?

**Comment by Dum4G** [comment]
>on what authority are you using to claim it is propagandist?
Well, common sense. Some channels simply show you official statements, claims and agenda, that's something that every self-respecting country has. But then there are channels where hosts and guests basically spread their ideology and their vision for every situation that curently goes on the way they like, their choices of what to cover and how are actually very predictable and thoroughly choosen, I would describe it as a tunnel vision television. 
>I think this thread is looking for the term "government organ"
Ironically, I would consider some overseas channels as politician, spreading basically untrust to the governments and someone elses agenda

**Comment by AntiPontifex** [comment]
Fair enough. With that baggage out of the way, so to speak, warning or category?

**Comment by Dum4G** [comment]
Any final decision?

**Comment by AntiPontifex** [comment]
My vote: warning

**Comment by BellezaEmporium** [comment]
I agree with the "political-affairs" category.

File: data/categories.csv (hunk 1 above)
Comment (BellezaEmporium) [reply data/categories.csv:21]: I would say that an underscore would be more fitting, rather than a hyphen.

**Pull request closed without merging**